sagemaker-clarify/fairness_and_explainability/bias_detection_with_predicted_label_and_facet_datasets.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>To keep the notebook clean, can we clear all cell outputs? Not only is it cleaner, but it is also less confusing for customers if the model and data processing is somewhat stochastic and they don't understand that and are trying to get the same results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3753/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>